modules/user_management.js

```diff
@@ -74,6 +74,74 @@ class UserManagement {
         }]
       }
     })
+
+    // Add a single literature quiz point to a user.
+    dispatcher.command('addpoint', this._guild.id, (authorMember, tokenized) => {
+      util.requireRole(authorMember, this._managementRole)
```

Comment:
Just to be clear: this is mods, not quiz masters.

Reply:
Yup, intended to be for mods and not quiz masters.